test/performance/shmem_perf_suite/shmem_bw_atomics_perf.c

```diff
@@ -342,52 +341,37 @@
             break;                                                             \
         }                                                                      \
         if(snode) {                                                            \
-            calc_and_print_results(end, start, len, metric_info);             \
+            calc_and_print_results(end, start, metric_info->start_len, metric_info);             \
         }                                                                      \
     } while(0)
 
-#define NUM_INC 100
-
-static const char * atomic_op_names [] = { "fetch", "set", "cswap", "swap", "finc", "inc",
-                                           "fadd", "add", "fand", "and", "for", "or",
-                                           "fxor", "xor" };
+static const char * atomic_op_names [] = { "fetch", "cswap", "swap", "finc", 
```

Comment:
Same comment as above, I think we can move this to `common.h` and doing so will ensure it's kept in sync if the enumeration is ever updated.